packages/presentation/lib/src/pages/movie_details_page/bloc/movie_details_bloc.dart

```diff
@@ -39,12 +45,22 @@ class _MovieDetailsBloc extends BlocImpl<BaseArguments, DetailsData>
           crewAndCast: peoplesToCrewUiMapper(peopleWithImage),
         ),
       );
-    } on AppException catch (e) {}
+    } on AppException catch (e) {
+      print(e.message);
+    }
   }
 
   @override
   void goBack() => appNavigator.pop();
 
+  @override
+  void share(BuildContext context) async {
```

Comment:
https://stackoverflow.com/questions/58505388/is-it-ok-to-pass-buildcontext-to-bloc

Reply:
agree =)